### DISCUSSION TITLE
Is a watch mode on the build command already supported?

### DISCUSSION THREAD

**Discussion by nckrtl** [maizzle/framework#1350] [Ideas]
Iv'e got Maizzle integrated in a Laravel app. The templates being built are exported to blade.php but as I use blade tags within the maizzle I can't preview the templates properly with the maizzle dev server. I must run the production build and then preview the exported files within my Laravel app to view them properly.

It would be nice if there is a watch option on the build command like --watch. This would trigger a build on any file change within Maizzle. Is this already possible? I basically have a PR ready to add the functionality but wanted to double check if such a watch mode might already be supported.

**Comment by cossssmin** [comment]
I think you might be able to use `serve` with a specific Maizzle config, i.e. `maizzle serve production` will use `config.production.js`.

Integrating with Laravel will be easier once we have the Vite plugin ready, it's currently under development.

**Comment by nckrtl** [comment]
That was exactly what I was looking for. I Wasn't aware you can append `production` to the serve command. Thanks for quick reply!

**Reply by MaartenJC** [reply]
Maizzle looks amazing, but i'm having the same problem.
But just changing this in package.json:
`"dev": "maizzle serve",`
to this
`"dev": "maizzle serve production",`
doesn't work.

I'm running version 5.0.8.
Refresh is working, but it is not building.
What am i doing wrong?

**Reply by nckrtl** [reply]
If I remember correctly I thought it was working, but turned out it wasn't the case. Been a while since I worked on the project using maizzle will try and verify later this week.

**Reply by cossssmin** [reply]
In 5.x `serve` does not build (as in, ouput files to disk). It just starts a local dev server.

**Reply by MaartenJC** [reply]
Thanks for the quicks response. So i got it working by installing chokidar-cli and concurrently and changing the package.json to this:
```
{
  "private": true,
  "type": "module",
  "scripts": {
    "dev": "concurrently \"maizzle serve\" \"chokidar \\\"emails/**/*.html\\\" -c \\\"maizzle build production\\\"\"",
    "build": "maizzle build production"
  },
  "dependencies": {
    "@maizzle/framework": "latest",
    "tailwindcss-preset-email": "latest"
  },
  "devDependencies": {
    "chokidar-cli": "^3.0.0",
    "concurrently": "^9.1.2"
  }
}
```
So this is the only option then?
